Mage.Sets/src/mage/cards/a/AethericAmplifier.java

```diff
@@ -0,0 +1,143 @@
+package mage.cards.a;
+
+import java.util.Set;
+import java.util.UUID;
+import java.util.stream.Collectors;
+
+import mage.abilities.Ability;
+import mage.abilities.Mode;
+import mage.abilities.common.ActivateAsSorceryActivatedAbility;
+import mage.abilities.costs.common.TapSourceCost;
+import mage.abilities.costs.mana.GenericManaCost;
+import mage.abilities.effects.OneShotEffect;
+import mage.abilities.mana.AnyColorManaAbility;
+import mage.cards.CardImpl;
+import mage.cards.CardSetInfo;
+import mage.constants.CardType;
+import mage.constants.Outcome;
+import mage.counters.Counter;
+import mage.counters.CounterType;
+import mage.game.Game;
+import mage.game.permanent.Permanent;
+import mage.players.Player;
+import mage.target.TargetPermanent;
+
+/**
+ * @author sobiech
+ */
+public final class AethericAmplifier extends CardImpl {
+
+    public AethericAmplifier(UUID ownerId, CardSetInfo setInfo) {
+        super(ownerId, setInfo, new CardType[]{CardType.ARTIFACT}, "{3}");
+
+        // {T}: Add one mana of any color.
+        this.addAbility(new AnyColorManaAbility());
+
+        // {4}, {T}: Choose one. Activate only as a sorcery.
+        // * Double the number of each kind of counter on target permanent.
+        final Ability ability = new ActivateAsSorceryActivatedAbility(new AethericAmplifierDoublePermanentEffect(), new GenericManaCost(4));
```

Comment:
Master updated by 06f24515cc4ffade9d23c1b99bf1d28fe2e64826, so you can use `ActivateAsSorceryActivatedAbility` with additional method withShowActivateText:

![shot_250208_233421](https://github.com/user-attachments/assets/8ceca68c-6b6c-4dc9-b3ef-82eee49eca1f)